src/celestia/qt/qtappwin.cpp

```diff
@@ -136,6 +136,47 @@ class AppAlerter : public CelestiaCore::Alerter
     QWidget* parent;
 };
 
+class FPSActionGroup
+{
+    QActionGroup *m_group;
+    std::map<int, QAction*> m_map;
```

Comment:
better `m_actions`, as m_map is meaningless name
the same for m_custom and m_last.

The should be self documenting, e.g. customFPSAction, lastFPS or smth similar.

Reply:
Indeed. I use short names to avoid lot's of typing... ;-)